test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -3931,24 +3931,22 @@ def log_lines(self, logtype, id=None, n=50, tail=True, starttime=None,
                 dayend = time.strftime("%Y%m%d", time.localtime(endtime))
                 firstday_obj = datetime.datetime.strptime(daystart, '%Y%m%d')
                 lastday_obj = datetime.datetime.strptime(dayend, '%Y%m%d')
+                if logtype == 'accounting':
+                    logdir = os.path.join(self.pbs_conf['PBS_HOME'],
+                                          'server_priv', 'accounting')
+                    sudo = True
+                elif (isinstance(self, Scheduler) and
```

Comment:
please add a comment why you need to explicitly check for scheduler logs.

Reply:
forgot to mention this. This is added to support multi-scheduler tests, so in that case path will be read from scheduler attributes.

Reply:
Ah thanks. please add a comment about it. otherwise it looks good.